### PR TITLE
delete code cycling images

### PR DESCRIPTION
**Pull request by Much4cho**

### CODE REVIEW

File: src/app/main-page/main-page.component.html [hunk 1]
```diff
@@ -8,16 +8,9 @@ <h2>by Adela Jaworowska</h2>
...
-    <img src="./assets/img/{{ images[imageIndex + 2] }}" />
-    <img src="./assets/img/{{ images[imageIndex + 3] }}" />
-    <img src="./assets/img/{{ images[imageIndex + 4] }}" />
-    <img src="./assets/img/{{ images[imageIndex + 5] }}" />
-    <img src="./assets/img/{{ images[imageIndex + 6] }}" />
-    <img src="./assets/img/{{ images[imageIndex + 7] }}" />
-    <img src="./assets/img/{{ images[imageIndex + 8] }}" />
-  </div>
+  <ng-container *ngFor="let image of images; index as i">
+    <img src="./assets/img/{{ image }}" *ngIf="i < 10"/>
```
Comment (aszulc97) [inline src/app/main-page/main-page.component.html:12]: I assume 10 is an amount of images. Can we do it in a dynamic way?

Comment (Much4cho) [reply src/app/main-page/main-page.component.html]: done

**Review by aszulc97 — APPROVED** [review]
you go gurl

**Comment by adelajaworowska** [comment]
Why the label is invalid?

**Pull request merged**